examples/azure-functions-v4/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@examples/azure-functions-v4",
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "main": "./server/handler.js",
```

Comment:
should we mention about the importance of this link inside the README "how it works" ?

Reply:
We could highlight it, absolutely!
It's also true that the ones that will use this should have some understanding about the model, that was my thought when writing the docs